examples/ts-without-babel/.gitignore

```diff
@@ -0,0 +1,23 @@
+# See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
```

Comment:
I don't think we need another gitignore here?

Reply:
Deleted gitignore.